### PR TITLE
Kobo only has mounts in /mnt/

### PR DESCRIPTION
**Pull request by BloodRagg**
I had some other mounts popping up
This seems like a good fix. (tested it)

### CODE REVIEW

**Comment by Frenzie** [comment]
Could you please give some more about your problem? Right now it's a bit mysterious. ;-)

```
# df -h
Filesystem                Size      Used Available Use% Mounted on
/dev/root               247.9M    148.6M     99.3M  60% /
none                     16.0M      4.0K     16.0M   0% /tmp
none                    248.4M      8.0K    248.4M   0% /dev
none                     16.0K      4.0K     12.0K  25% /var/lib
none                     16.0K         0     16.0K   0% /var/log
none                    128.0K      8.0K    120.0K   6% /var/run
/dev/mmcblk0p3            3.2G      2.8G    382.7M  88% /mnt/onboard
```

**Comment by NiLuJe** [comment]
Yeah, this doesn't change a thing, as only mmcblk* stuff (internal/external) will get mounted to /mnt/onboard or /mnt/sdcard

**Comment by BloodRagg** [comment]
I have some bind mounts on rootfs that get listed:
Filesystem                Size      Used Available Use% Mounted on
/dev/root               247.9M    142.7M    105.2M  58% /opt
/dev/loop0               31.7M     18.9M     12.8M  60% /
none                     16.0M      8.0K     16.0M   0% /tmp
none                    250.9M     12.0K    250.9M   0% /dev
none                     16.0K         0     16.0K   0% /lib/modules
none                     16.0K         0     16.0K   0% /var/log
none                    128.0K      8.0K    120.0K   6% /var/run
none                    128.0K         0    128.0K   0% /mnt
/dev/loop0               31.7M     18.9M     12.8M  60% /lib/modules/2.6.35.3-850-gbc67621+
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /mnt/onboard
/dev/mmcblk1p1           29.7G     22.9G      6.8G  77% /mnt/sd
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/settings.reader.lua
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/history.lua
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/cache
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/clipboard
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/data/dict
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/data/tessdata
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/history
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/ota
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/screenshots
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/settings
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/styletweaks

File: plugins/systemstat.koplugin/main.lua [hunk 1]
```diff
@@ -16,8 +16,10 @@ local SystemStat = {
 }
 
 function SystemStat:init()
-    if Device:isCervantes() or Device:isKobo() or Device:isPocketBook() then
+    if Device:isCervantes() or Device:isPocketBook() then
         self.storage_filter = "mmcblk"
+    elseif Device:isKobo() then
+        self.storage_filter = "/mnt/"
```
Comment (NiLuJe) [inline plugins/systemstat.koplugin/main.lua]: ```self.storage_filter = = ' /mnt/onboard$' -e ' /mnt/sdcard$'```

Comment (NiLuJe) [reply plugins/systemstat.koplugin/main.lua]: Assuming we only care about default storage locations, that is ;).

Comment (Frenzie) [reply plugins/systemstat.koplugin/main.lua]: Seems a bit too exclusive to me. :-P

Comment (BloodRagg) [reply plugins/systemstat.koplugin/main.lua]: Kobo user data is only mounted in /mnt, so i figured to set the filter on /mnt/.
Its sufficient for a standard Kobo and filters out any other mounts (loop devices, binds)

Comment (NiLuJe) [reply plugins/systemstat.koplugin/main.lua]: I'd keep the space, at least, to match on mountpoint and not blockdevice, on the off-chance the syntax gets wonky one day ;).

**Comment by pazos** [comment]
> /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/settings.reader.lua
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/history.lua
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/cache
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/clipboard
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/data/dict
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/data/tessdata
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/history
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/ota
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/screenshots
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/settings
/dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/styletweaks

why not simply use a single tmpfs for /opt/ram/koreader and generate soft links to the particular files/folders?

File: plugins/systemstat.koplugin/main.lua (hunk 1 above)
Comment (BloodRagg) [reply plugins/systemstat.koplugin/main.lua]: Keeping the space =P

**Comment by BloodRagg** [comment]
> > /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/settings.reader.lua
> > /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/history.lua
> > /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/cache
> > /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/clipboard
> > /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/data/dict
> > /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/data/tessdata
> > /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/history
> > /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/ota
> > /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/screenshots
> > /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/settings
> > /dev/mmcblk0p3           28.8G      7.2G     20.1G  26% /opt/ram/koreader/styletweaks
> 
> why not simply use a single tmpfs for /opt/ram/koreader and generate soft links to the particular files/folders?

Softlinks get overwritten on update, tar doesn't care.
I hope koreader consolidates it into one profile some day,  then its only one bind.

**Comment by Frenzie** [comment]
Not entirely sure what I think, but I suppose it doesn't change behavior in default setups. :-)

**Pull request merged**